nightlies.conf

```diff
@@ -27,7 +27,8 @@ apt=protobuf-compiler python3-protobuf
 slack=browsers
 
 [oflatt/eggcc]
-slack=eggcc
+slack=eggcc-notify
```

Comment:
No do not change this. This is not a channel name, it’s a key into a secrets file.

Reply:
Oops, okay